config/routes.rb

```diff
@@ -6,5 +6,6 @@
     get ':organization_id/uploads/last', to: 'uploads#last', constraints: { organization_id: %r{[^\/]+} }
     get ':organization_id/uploads/file', to: 'uploads#download_file', constraints: { organization_id: %r{[^\/]+} }
     get 'accounts', to: 'accounts#index'
+    post 'toggle_auto_upload', to: 'uploads#toggle_auto_upload'
```

Comment:
```suggestion
    post 'auto_upload', to: 'uploads#auto_upload'
```
And the value will be passed as a prameter